scripts/versions

```diff
@@ -3,6 +3,7 @@ set -euo pipefail
 
 # Versions to be used
 declare -A VERSIONS=(
+    ["go"]=1.15
```

Comment:
Let's use 1.15.3 here

Can we also use the latest CircleCI machine image which uses 20.04: ubuntu-2004:202010-01

Reply:
Right. It's too bad we have to specify a minor version.

Switched the image to 20.04 as well